tuna/find_db.py

```diff
@@ -70,6 +71,8 @@ def get_query(self, sess, fdb_obj, session_id):
     if self.solver:
       query = query.filter(fdb_obj.solver == self.solver)
 
+    self.logger.info("fdb query %s-%s-%s-%s", session_id, self.config, self.solver, self.opencl)
```

Comment:
Don't know why the linter didn't complain, but it usually wants f-strings

Reply:
Sure, I can swap this to an f-string.

Reply:
Remembered that this is because of the logger function. The logger prefers this syntax.

Reply:
oh I see